src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -382,6 +383,17 @@ def stop_servers(self):
         """
         error_list = []
         if self.hostfile_servers:
+            # Reset the nvme storage
+            if self.nvme_parameter == "nvme":
+                self.multi_log("Resetting NVMe storage on the servers")
+                try:
+                    server_utils.storage_reset(self.hostlist_servers)
```

Comment:
I'm on the fence about this one.  On the one hand I'm a fan of tests cleaning up their mess.  But on the other hand, it can be very useful for tests not to alter their final state to facilitate test post-mortem in the case of debugging.

This step absolutely should be done to storage on all nodes prior to a test starting, yes?  If so, I'd be inclined to err on the side of facilitating the post-mortem.

What are your thoughts @phender?

Reply:
I am inclined to agree.   This seems more like a `setUp()` action.   Shouldn't we fix it in a separate PR?